packages/autocomplete-plugin-tags/src/createTagsPlugin.tsx

```diff
@@ -0,0 +1,173 @@
+/** @jsx createElement */
+import { BaseItem, PluginSubscribeParams } from '@algolia/autocomplete-core';
+import {
+  AutocompletePlugin,
+  AutocompleteSource,
+  AutocompleteState,
+} from '@algolia/autocomplete-js';
+
+import { createTags, OnTagsChangeParams } from './createTags';
+import type { BaseTag, Tag } from './types';
+
+type OnChangeParams<TTag> = PluginSubscribeParams<any> &
+  OnTagsChangeParams<TTag>;
+
+type TagsPluginData<TTag> = {
+  /**
+   * Returns the current list of tags.
+   *
+   * @link https://www.algolia.com/doc/ui-libraries/autocomplete/api-reference/autocomplete-plugin-tags/createtagsplugin/#param-tags
+   */
+  readonly tags: Array<Tag<TTag>>;
```

Comment:
We don't use `readonly` yet in the codebase, so can we remove it for now and introduce them later?

Reply:
Handled in 4045417.